### DISCUSSION TITLE
How to add my custom layout made with MSKLC?

### DISCUSSION THREAD

**Discussion by ghost** [DreymaR/BigBagKbdTrixPKL#42] [Q&A] [answered]
I've tried many things but nothing worked, didn't find a guide on how to do this, any ideas?

**Comment by DreymaR** [comment, marked as answer]
At the moment you can't add MSKLC layouts directly. You have to remake them as EPKL layouts. In the future I hope to make an import module, but that's still a way off.

Aldo39 has a layout conversion tool but he hasn't added EPKL as one of his formats. So no luck there either, for now.

So the closest you get to a guide is the guide to making variants in the Readme. And also, look at other layouts and how they're made.
